README.md

```diff
@@ -41,7 +41,8 @@ python manage.py createsuperuser
 python manage.py runserver
 ```
 
-For detailed installation and setup docs, see [docs.wagtail.org](https://docs.wagtail.org/).
+Note: Above commands are the implementation of Wagtail, NOT setup guide.
```

Comment:
Please can this unrelated change be removed from this PR - I think you have already covered this suggestion in a seperate PR.

Reply:
yes, you're right

It was my bad, I've reverted this specific commit.